From 2f2cd7a8657ebc7cdbddabe1166cbdcb94d66995 Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Wed, 11 Dec 2024 07:32:04 +0100
Subject: [PATCH] man/systemd-cryptenroll: sort --fido2-credential-algorithm
 after --fido2-device

And also fix a typo.

(cherry picked from commit 62b7b70bb78f85198509b3327cb29b83ed9218ec)
---
 man/systemd-cryptenroll.xml | 32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/man/systemd-cryptenroll.xml b/man/systemd-cryptenroll.xml
index 83f8b2d8a1..f3c23adfd0 100644
--- a/man/systemd-cryptenroll.xml
+++ b/man/systemd-cryptenroll.xml
@@ -384,24 +384,9 @@
   <refsect1>
     <title>FIDO2 Enrollment</title>
 
-    <para>The following options are understood that may be used to enroll PKCS#11 tokens:</para>
+    <para>The following options are understood that may be used to enroll FIDO2 tokens:</para>
 
     <variablelist>
-      <varlistentry>
-        <term><option>--fido2-credential-algorithm=<replaceable>STRING</replaceable></option></term>
-        <listitem><para>Specify COSE algorithm used in credential generation. The default value is
-        <literal>es256</literal>. Supported values are <literal>es256</literal>, <literal>rs256</literal>
-        and <literal>eddsa</literal>.</para>
-
-        <para><literal>es256</literal> denotes ECDSA over NIST P-256 with SHA-256. <literal>rs256</literal>
-        denotes 2048-bit RSA with PKCS#1.5 padding and SHA-256. <literal>eddsa</literal> denotes
-        EDDSA over Curve25519 with SHA-512.</para>
-
-        <para>Note that your authenticator may choose not to support some algorithms.</para>
-
-        <xi:include href="version-info.xml" xpointer="v251"/></listitem>
-      </varlistentry>
-
       <varlistentry>
         <term><option>--fido2-device=<replaceable>PATH</replaceable></option></term>
 
@@ -428,6 +413,21 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--fido2-credential-algorithm=<replaceable>STRING</replaceable></option></term>
+        <listitem><para>Specify COSE algorithm used in credential generation. The default value is
+        <literal>es256</literal>. Supported values are <literal>es256</literal>, <literal>rs256</literal>
+        and <literal>eddsa</literal>.</para>
+
+        <para><literal>es256</literal> denotes ECDSA over NIST P-256 with SHA-256. <literal>rs256</literal>
+        denotes 2048-bit RSA with PKCS#1.5 padding and SHA-256. <literal>eddsa</literal> denotes
+        EDDSA over Curve25519 with SHA-512.</para>
+
+        <para>Note that your authenticator may choose not to support some algorithms.</para>
+
+        <xi:include href="version-info.xml" xpointer="v251"/></listitem>
+      </varlistentry>
+
       <varlistentry>
         <term><option>--fido2-salt-file=<replaceable>PATH</replaceable></option></term>
 
